integration_tests/Cargo.toml

```diff
@@ -7,7 +7,7 @@ althea_proto = "0.9"
 prost = "0.13"
 prost-types = "0.13"
 tonic = "0.12"
-deep_space = {version = "=2.28.2", features = ["althea", "ethermint"]}
+deep_space = {version = "=2.28.1", features = ["althea", "ethermint"]}
```

Comment:
why downgrade?

Reply:
I yanked that version, it's just a type url change to two protos but the change was incorrect.